duck/ornithology.py

```diff
@@ -9,27 +9,58 @@
 ANY = mock.ANY
 
 
-class Instance(object):
-    """A class that checks equality based on type.
+class Predicate(object):
+    """An object considered to be equal to anything that passes its predicate
+    function.
 
-    Instances of Instance have an .__eq__ method that returns
-    True if and only if the other comparison object is an
-    instance of the provided class (as defined by isinstance).
+    Args:
+        predicate (Callable): The predicate to use to test.
```

Comment:
`predicate (Callable -> bool): The predicate to use as a test. It should return a boolean.`